openvas-smb compiling fails

Hi,

I'm getting errors while compiling openvas-smb

OS: Ubuntu Server 25.04 arm64

```
/root/source/openvas-smb-22.5.3/samba/auth/credentials/credentials_krb5.c:309:20: error: implicit declaration of function ‘gss_krb5_import_cred’; did you mean ‘gss_import_cred’? [-Wimplicit-function-declaration]
  309 |         maj_stat = gss_krb5_import_cred(&min_stat, ccache->ccache, NULL, NULL,
      |                    ^~~~~~~~~~~~~~~~~~~~
      |                    gss_import_cred
/root/source/openvas-smb-22.5.3/samba/auth/credentials/credentials_krb5.c: In function ‘cli_credentials_set_client_gss_creds’:
/root/source/openvas-smb-22.5.3/samba/auth/credentials/credentials_krb5.c:360:20: error: implicit declaration of function ‘gss_krb5_copy_ccache’ [-Wimplicit-function-declaration]
  360 |         maj_stat = gss_krb5_copy_ccache(&min_stat,
      |                    ^~~~~~~~~~~~~~~~~~~~
gmake[2]: *** [samba/CMakeFiles/samba-static.dir/build.make:1473: samba/CMakeFiles/samba-static.dir/auth/credentials/credentials_krb5.c.o] Error 1
gmake[2]: *** Waiting for unfinished jobs....
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -154,7 +154,8 @@ if(${GSSAPI_VERSION} VERSION_LESS "7.5.0")
 	add_definitions(-DOLD_HEIMDAL)
 endif()
 set (CMAKE_C_FLAGS "-fPIC -O2 -D_FORTIFY_SOURCE=2 -fstack-protector ${CMAKE_C_FLAGS}")
-set (CMAKE_C_FLAGS "-Wno-unused-result -fno-strict-aliasing ${CMAKE_C_FLAGS}")
+set (CMAKE_C_FLAGS "-Wno-incompatible-pointer-types -Wno-implicit-function-declaration -Wno-unused-result -fno-strict-aliasing ${CMAKE_C_FLAGS}")
+
 set (LINKER_HARDENING_FLAGS     "-Wl,-z,relro -Wl,-z,now")
 
 ## Version

